### DISCUSSION TITLE
Applications on same VPS as Vito

### DISCUSSION THREAD

**Discussion by Danny159** [vitodeploy/vito#48] [General]
Hey!

I have installed Vito onto a clean VPS. All works fine, logged in ok!

I want to use the same VPS as the application server. However when I try and add the same server I get the following error;

> The installation has been failed on step: Installing ufw (100%)

In the Logs I get the following;

> sudo: ufw: command not found

Any help would be appreciated.

**Comment by saeedvaziry** [comment]
Vito is not designed to install applications on itself. Otherwise, it will crash.

You can use it to provision other servers.

If having 1 separate instance for Vito is not an option for you then you can consider using Vito locally to manage your remote servers

https://vitodeploy.com/introduction/installation.html#install-locally

**Comment by Danny159** [comment]
Ahh that would explain it then.

What in the minimum VPS requirement to run Vito?

**Reply by saeedvaziry** [reply]
At least 1 GB memory and 1 CPU
